test/Graphics/ConvexShape.test.cpp

```diff
@@ -42,4 +43,31 @@ TEST_CASE("[Graphics] sf::ConvexShape")
         convex.setPoint(0, {3, 4});
         CHECK(convex.getPoint(0) == sf::Vector2f(3, 4));
     }
+
+    SUBCASE("Reasonable geometric center")
+    {
+        sf::ConvexShape convex;
+        for (unsigned i = 2; i < 10; ++i)
+        {
+            sf::CircleShape circle(4.f, i);
+            convex.setPointCount(i);
+            for (unsigned j = 0; j < i; ++j)
+            {
+                convex.setPoint(j, circle.getPoint(j));
+            }
+            CHECK(convex.getGeometricCenter() == Approx(circle.getGeometricCenter()));
+        }
+    }
+
+    SUBCASE("Edge case geometric centers")
+    {
+        sf::ConvexShape convex2(1);
```

Comment:
Instead of having `convex2` and `convex3` (which btw did you mean to start counting at 2?) can there be two SUBCASEs within this SUBCASE that describe what each one is testing? Presumably these two tests are covering different edge cases so lets spell out what those edge cases are.